jax/_src/numpy/lax_numpy.py

```diff
@@ -7443,9 +7466,33 @@ def cross(a, b, axisa: int = -1, axisb: int = -1, axisc: int = -1,
   return moveaxis(c, 0, axisc)
 
 
-@util.implements(np.kron)
 @jit
 def kron(a: ArrayLike, b: ArrayLike) -> Array:
+  """Compute the Kronecker product of two input arrays.
```

Comment:
I'm not sure how familiar the Kronecker product is. We may want to describe the computation it does

Reply:
Kronecker product description is added.